README.md

```diff
@@ -28,3 +28,61 @@ Step 7: yarn run start:dev
 - [Node.js](https://nodejs.org/en/) - Tested against LTS
 - NPM (installed with Node.js) - Works with versions 2 and 3
 - [Redis server](http://redis.io/download) running on the default port
+
+
+### Database setup and integration
+
+If this is a first-time install get postgres running on the default port and setup a new, empty local database called `acrs`.  
```

Comment:
I think installing postgres is something that should go in the install prerequisites section

Reply:
Moved